packages/webcomponents/src/api/services/payout.service.ts

```diff
@@ -23,9 +22,8 @@ export class PayoutService implements IPayoutService {
     params: any,
     apiOrigin?: string
   ): Promise<IApiResponseCollection<IPayout[]>> {
-
     if (!apiOrigin) {
-      apiOrigin = config.proxyApiOrigin;
+      apiOrigin = PROXY_API_ORIGIN;
```

Comment:
If we can remove the if/else block that sets iframe origin in the payment.service.ts file, then I guess we can do the same thing a few times in this file. 🤔